enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -88,6 +90,61 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
         </div>
       </div>
     </div>
-
+    <div id="course-details-modal" class="modal" aria-hidden="true">
+      <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+        <button class="modal-close-button" id="modal-close-button" aria-label="${_('Close')}">${_('Close')}</button>
+        <header class="wrapper">
+          <div class="image">
+            <img src="${ course_image_uri }" alt="${ course_name }"/>
+          </div>
+          <div class="details">
+            <h1 id="modal-header-text">${ course_name }</h1>
+            <p class="short-description">${ course_short_description }</p>
+            <div class="enterprise">
+              <img src="${static.url('images/logo.png')}" alt="${ course_organization }" />
```

Comment:
This still isn't correct. This is just a link to the platform logo. We need to use the logo of the _course's creating organization_.

You can fetch this information via the Organizations API with a URL like this:

```
http://localhost:8000/api/organizations/v0/organizations/edX/
```

where edX is the value of the `org` key in `course_details`.

That'll return a value like the following:

```json
{
    "id":1,
    "created":"2017-05-11T07:02:14.250525Z",
    "modified":"2017-05-11T07:02:14.308697Z",
    "name":"edX",
    "short_name":"edX",
    "description":"",
    "logo":"http://localhost:8000/media/organization_logos/Screen_Shot_2017-05-20_at_4.13.02_PM.png",
    "active":true
}
```

You can then use the details fetched using that API to select the appropriate image URL. I'd also set the alt text based on the value of the `name` key in the response.